python/tskit/trees.py

```diff
@@ -5788,6 +5801,52 @@ def individuals_flags(self):
         """
         return self._individuals_flags
 
+    @property
+    def individuals_metadata(self):
+        """
+        Efficient access to the ``metadata`` column in the
+        :ref:`sec_individual_table_definition` as a structured numpy array.
+        The returned dtype will depend on the metadata schema used. Only a subset
+        of `struct` metadata schemas are supported.
+        See :ref:`sec_structured_array_metadata` for more information.
+        """
+        return self.table_metadata_schemas.individual.structured_array_from_buffer(
+            self._individuals_metadata
+        )
+
+    @property
+    def individuals_metadata_offset(self):
```

Comment:
I'm not sure we want to offer this array in the high-level API. It may just be confusing. I would pull this out of the current PR, as it's orthogonal

Reply:
Offset accessors removed in https://github.com/tskit-dev/tskit/pull/3098/commits/b74b824a2a20a245542bf7600e5ae2a82f73bbeb